packages/web-components/src/components/va-banner/test/va-banner.spec.ts

```diff
@@ -8,7 +8,7 @@ describe('va-banner', () => {
     cmp.el.innerHTML = 'This is test content';
     cmp.showClose = true;
     expect(cmp.dismissedBanners).toEqual([]);
-    cmp.dismiss();
+    cmp['dismiss']();
```

Comment:
Changed syntax due to type errors on build.

Reply:
It's super minor, but could you update this to use double quotes instead of single quotes so it's consistent with all of the changes below?